rcongui/src/components/Scoreboard/utils.js

```diff
@@ -1,12 +1,25 @@
 export const UNKNOWN_MAP_NAME = 'unknown';
 
+export const SKIRMISH_MAPS = [
```

Comment:
do we need this array? 🤔

Reply:
No, I think I added that when I was originally messing around with this and forgot all about it lol.